packages/slice-machine/src/models/libraries/context.tsx

```diff
@@ -24,23 +24,25 @@ const LibraryHandler: React.FunctionComponent<LibraryHandlerProps> = ({
   remoteSlices,
   env,
 }) => {
-  const models: ReadonlyArray<LibraryState> = libraries.map((lib) => {
-    return {
-      name: lib.name,
-      isLocal: lib.isLocal,
-      components: lib.components.map((component) =>
-        useModelReducer({
-          slice: component,
-          mockConfig: SliceMockConfig.getSliceMockConfig(
-            env.mockConfig,
-            lib.name,
-            component.infos.sliceName
-          ),
-          remoteSlice: remoteSlices?.find((e) => e.id === component.model.id),
-        })
-      ),
-    };
-  });
+  const models: ReadonlyArray<LibraryState> | undefined = libraries?.map(
```

Comment:
same

Reply:
If the libraries returned are not defined, this "models" which will be in the provider should be undefined as well.